pulp_smash/tests/puppet/api_v2/test_sync_publish.py

```diff
@@ -76,6 +82,40 @@ def _gen_repo():
         'notes': {'_repo-type': 'puppet-repo'},
     }
 
+class _CleanUpTestCaseMixin(object):
+    """Provide functions to clean up database and filesystem of Pulp"""
+
+    @classmethod
+    def reset_database(cls, server_config):
```

Comment:
This method references both `cls.cfg` and `server_config`. How about turning this into a static method?

```python
@staticmethod
def reset_database(server_config):
    ...
```

This makes it more obvious what's going on.